src/magellan_firmware/firmware/magellan_controller/HeartbeatLED.cpp

```diff
@@ -1,23 +1,21 @@
 #include "HeartbeatLED.h"
 #include "Arduino.h"
 
-HeartbeatLED::HeartbeatLED() :
-    heartbeat_led_rate_(HEARTBEAT_DISABLED_HZ),
-    heartbeat_led_state_(false) {
+HeartbeatLED::HeartbeatLED() : heartbeat_led_rate_(HEARTBEAT_DISABLED_HZ), heartbeat_led_state_(false) {
```

Comment:
Can we make it so that each object in the initializer list is on its own line?

Reply:
Fixed